build-setup/windows.md

```diff
@@ -233,16 +233,40 @@ git submodule sync
 git submodule update --init --recursive
 ```
 
+In `caffe2/utils/CMakeLists.txt` delete the following line (line 4):
+
+```
+    utils/threadpool/pthreadpool-cpp.cc
+```
+
+In `torch/CMakeLists.txt` remove the guards around `onnx_library`, i.e. change lines 99-101 from:
+
+```
+    if(BUILD_TEST)
+      list(APPEND TORCH_PYTHON_LINK_LIBRARIES onnx_library)
+    endif(BUILD_TEST)
+```
+
+to:
+
+```
+    list(APPEND TORCH_PYTHON_LINK_LIBRARIES onnx_library)
+```
```

Comment:
This one is still a problem in the current master branch code of PyTorch - see https://github.com/pytorch/pytorch/issues/45060#issuecomment-766332179